frontend/src/components/UserSearch.js

```diff
@@ -27,11 +51,17 @@ const UserSearch = () => {
           <b>Reminder:</b> Search for participant profile prior to creating a
           new profile
         </Typography>
-        <form className="participant-search__form">
+        <form className="participant-search__form" onSubmit={handleSubmit}>
           <FormGroup className="participant-search__input">
             <FormControl>
               <InputLabel htmlFor="user_id">User ID</InputLabel>
-              <Input id="user_id" name="user_id" value="" required />
+              <Input
+                id="user_id"
+                name="user_id"
+                value={participantsStore.params.userId}
+                onChange={e => participantsStore.setUserIdParam(e.target.value)}
```

Comment:
This looks good, I'm hoping it fixes the typing/input issue we were having in this component.

Reply:
indeed it does